include/z64save.h

```diff
@@ -177,11 +177,11 @@ typedef struct {
     /* 0x13E0 */ u8 seqId;
     /* 0x13E1 */ u8 natureAmbienceId;
     /* 0x13E2 */ u8 buttonStatus[5];
-    /* 0x13E7 */ u8 unk_13E7; // alpha related
-    /* 0x13E8 */ u16 unk_13E8; // alpha type?
-    /* 0x13EA */ u16 unk_13EA; // also alpha type?
-    /* 0x13EC */ u16 unk_13EC; // alpha type counter?
-    /* 0x13EE */ u16 unk_13EE; // previous alpha type?
+    /* 0x13E7 */ u8 hudModeDimOnlyDisabledButtons; // for certain hud displays, will override button dim request and ensure the buttons are still visible
+    /* 0x13E8 */ u16 requestHudMode; // triggers the hud to change modes to the requested value. Reset to 0 when target is reached
+    /* 0x13EA */ u16 curHudMode; // current hud mode
+    /* 0x13EC */ u16 hudModeCounter; // number of frames in the transition to a new hud display. Used to step alpha
```

Comment:
Why not call this a timer? Also mentioning this ticks up in the comment, could be useful (it does tick up, right)

Reply:
Oh, to me "timer" means "ticks every frame", but yeah I can see how this would be a weird "timer" since it has no definite goal/end time maybe

I'm fine with "counter" then, it works too :)

Reply:
its something that people in general dont agree on-- I personally always use timer no matter the direction. to me it just means a frame counting in either direction

Its something we'll probably need to agree on at a larger scale and fix other usages

Reply:
where I come from, a counter is just an integer (an int32 specifically, implemented sometime after MZX was ported out of DOS)

Reply:
And sometimes they count down or up depending on the circumstances, too.